### DISCUSSION TITLE
Does Fan Control opens a CMD on startup?

### DISCUSSION THREAD

**Discussion by huangfanglong** [Rem0o/FanControl.Releases#935] [Q&A] [answered]
Hi all,

Just a quick question of concern, does Fan Control automatically open a cmd line on computer startup when you set it to start on computer startup? Just wanted to make sure if it's this or something else so I can further look into it if it's the latter.

Thanks

**Comment by Rem0o** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/14118956/178292685-c349a971-bdc9-4c03-9a57-97efc4c832c6.png)

**Reply by huangfanglong** [reply]
Thank you so much.
